Source/WebCore/platform/network/soup/NetworkStorageSessionSoup.cpp

```diff
@@ -80,6 +80,44 @@ NetworkStorageSession::~NetworkStorageSession()
     g_signal_handlers_disconnect_matched(m_cookieStorage.get(), G_SIGNAL_MATCH_DATA, 0, 0, nullptr, nullptr, this);
 }
 
+#if HAVE(COOKIE_CHANGE_LISTENER_API)
+void NetworkStorageSession::notifyCookieAdded(SoupCookie* newCookie)
+{
+    if (soup_cookie_get_http_only(newCookie))
+        return;
+
+    auto host = String::fromUTF8(soup_cookie_get_domain(newCookie));
+    for (auto& host : m_cookieChangeObservers.keys()) {
```

Comment:
The same variable name `host` is used here. Can the loop variable be made `const ref` afterwards?